packages/blade/src/components/Box/__tests__/Box.native.test.tsx

```diff
@@ -17,6 +17,7 @@ describe('<Box />', () => {
     );
     expect(toJSON()).toMatchInlineSnapshot(`
       <View
+        data-blade-component="box"
```

Comment:
does this works on native? IIRC from past, custom attributes were discarded by RN?

Reply:
It doesn't discard it during tests. So the react-test-renderer that's used during tests to generate these snapshots doesn't discard it.